src/features/addTweetActions.ts

```diff
@@ -194,36 +235,129 @@ export const maybeAddTweetActions = makeBTDModule(({settings, TD, jq}) => {
     navigator.clipboard.writeText(mediaUrls);
   });
 
+  modifyMustacheTemplate(TD, 'stream_item.mustache', (string) =>
+    string.replace(
+      'data-tweet-id="{{#getMainTweet}}{{id}}{{/getMainTweet}}"',
+      'data-tweet-id="{{#getMainTweet}}{{id}}{{/getMainTweet}}" data-tweet-user-id="{{#getMainUser}}{{id}}{{/getMainUser}}{{^getMainUser}}{{targetTweet.user.id}}{{/getMainUser}}"'
+    )
+  );
+
+  TD.mustaches['text/follow_action.mustache'] =
+    '{{^following}} {{_i}}Follow @{{screenName}}{{/i}} {{/following}} {{#following}} {{_i}}Unfollow @{{screenName}}{{/i}} {{/following}}';
+
+  modifyMustacheTemplate(TD, 'status/tweet_single.mustache', (string) =>
+    string.replace(
+      '{{#isFavorite}}is-favorite{{/isFavorite}}',
+      '{{#isFavorite}}is-favorite{{/isFavorite}} {{#getMainUser}}{{#following}}following{{/following}}{{/getMainUser}}'
+    )
+  );
+
+  modifyMustacheTemplate(TD, 'status/tweet_detail.mustache', (string) =>
+    string
+      .replace(
+        'data-tweet-id="{{#getMainTweet}}{{id}}{{/getMainTweet}}"',
+        'data-tweet-id="{{#getMainTweet}}{{id}}{{/getMainTweet}}" data-tweet-user-id="{{getMainUser.id}}"'
+      )
+      .replace(
+        '{{#isFavorite}}is-favorite{{/isFavorite}}',
+        '{{#isFavorite}}is-favorite{{/isFavorite}} {{#getMainUser}}{{#following}}following{{/following}}{{/getMainUser}}'
+      )
+  );
+
+  // Similar to `TD.services.TwitterStatus.prototype.setFavorite`
+  TD.services.TwitterUser.prototype.setFollowing = function (following: boolean) {
+    this.following = following;
+    const addedClass = following ? 'icon-following' : 'icon-follow';
+    const removedClass = following ? 'icon-follow' : 'icon-following';
+    jq(`[data-tweet-user-id="${this.id}"][data-account-key="${this.account.getKey()}"]`)
+      .find('.js-tweet')
+      .toggleClass('following', following)
+      .find('a[rel="follow"]')
+      .attr('title', TD.ui.template.render('text/follow_action', this))
+      .find('.js-icon-follow')
+      .addClass(addedClass)
+      .removeClass(removedClass);
+  };
+
+  const findPrefix: ((n: number, options: unknown) => string) | undefined =
+    mR.findFunction('findPrefix')[0];
+
+  if (findPrefix) {
+    TD.services.TwitterUser.prototype.prettyFollowersCountInTweetAction = function () {
+      return findPrefix(this.followersCount, {separator: '', decimals: 0});
+    };
+  }
```

Comment:
What happens in the templates if `findPrefix` is undefined? Does it crash?

Reply:
No, it doesn't crash. 
It just stops rendering `{{prettyFollowersCountInTweetAction}}` part in the template.